### DISCUSSION TITLE
Having a dedicated reverse proxy for a Realm

### DISCUSSION THREAD

**Discussion by alirezameskin** [keycloak/keycloak#12170] [Q&A]
Hello everyone

I was wondering whether a proxy could be assigned to a specific realm in Keycloak. for instance, instead of https://keycloak.domain/auth/realms/master/account/ I would like to use https://keycloak.domain/account/.

Best
Alireza

**Comment by pedroigor** [comment]
AFAIK, it is not possible due to how we create URLs on server side. There are a lot of places where the URLs returned by the server are not going to remove the path, like when doing redirects.

I think that is something @stianst wants to improve in our hostname SPI.
